lib/v2/gov/router.js

```diff
@@ -51,4 +51,5 @@ module.exports = function (router) {
     router.get('/sichuan/deyang/govpulicinfo/:countyName', require('./sichuan/deyang/govpulicinfo'));
     router.get('/taiyuan/rsj/:caty/:page?', require('./taiyuan/rsj'));
     router.get('/xuzhou/hrss/:category?', require('./xuzhou/hrss'));
+    router.get('/gzgov/:channel/:category', require('./gzgov/index'));
```

Comment:
- The domain name is `gz` please use that instead.
- Please help maintain entries in this file in alphabetical order